com/duke/tasks/Deadlines.java

```diff
@@ -0,0 +1,60 @@
+package com.duke.tasks;
+
+import java.time.LocalDate;
+
+public class Deadlines extends Task {
+    protected LocalDate date;
+    protected int time;
+
+    public Deadlines(String task, String dateAndTime) {
+        //date = 2019-12-02 1800
+        String[] dateAndTimeArr = dateAndTime.split(" ");
+//        // date = 'by Sunday'
+//        String date = task.substring(task.indexOf("/") + 1, task.length());
+//        date = com.duke.events.Task.reformatDate(date); // (by: Sunday)
+//
+//        // task = return book
+//        task = task.substring(0, task.indexOf("/") - 1);
+        this.task = task;
+        this.date = LocalDate.parse(dateAndTimeArr[0]);
+        this.time = Integer.parseInt(dateAndTimeArr[1]);
+    }
+
+    public Deadlines(String task, String dateAndTime, boolean done) {
+        String[] dateAndTimeArr = dateAndTime.split(" ");
+
+        this.task = task;
+        this.date = LocalDate.parse(dateAndTimeArr[0]);
+        this.time = Integer.parseInt(dateAndTimeArr[1]);
+        this.done = done;
+    }
+
+    @Override
+    public String toString() {
+        String doneIndicator = this.done ? "[✓]" : "[✗]";
```

Comment:
Would it be better to make "[✓]" and "[✗]" named constants to avoid using 'magic' values?